From: Simon McVittie <smcv@collabora.com>
Date: Mon, 18 Jan 2021 09:57:55 +0000
Subject: glslang.pc: Add missing libraries

-lglslang depends on -lMachineIndependent, and -lOGLCompiler depends on
-lGenericCodeGen.

v2 OGLCompiler was removed, so remove GenericCodeGen

Signed-off-by: Simon McVittie <smcv@collabora.com>

Philippe SWARTVAGHER <phil.swart@gmx.fr>, 2024-05-26: GenericCodeGen seems to be still required.
---
 glslang/glslang.pc.cmake.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/glslang/glslang.pc.cmake.in b/glslang/glslang.pc.cmake.in
index 921497e..8c49e0c 100644
--- a/glslang/glslang.pc.cmake.in
+++ b/glslang/glslang.pc.cmake.in
@@ -7,5 +7,5 @@
 Description: OpenGL and OpenGL ES shader front end and validator
 Requires:
 Version: @VERSION@
-Libs: -L${libdir} -lglslang -lOSDependent -lSPVRemapper
-Cflags: -I${includedir}
+Libs: -L${libdir} -lglslang -lMachineIndependent -lGenericCodeGen -lOSDependent -lSPVRemapper -lpthread
+Cflags: -I${includedir}
